README.md

```diff
@@ -160,6 +160,7 @@ Table of Contents
   * [Beeceptor](https://beeceptor.com) - Mock a rest API in seconds, fake API response and much more. Free 50 requests per day, public dashboard, open endpoints (anyone having link to the dashboard can view requests and responses).
   * [ScreenshotAPI.net](https://screenshotapi.net/) - Screenshot API use one simple API call to generate screenshots of any website. Build to scale and hosted on Google Cloud. Offers 100 free screenshots per month.
     * [shrtcode API](https://shrtco.de/docs) - Free URL Shortening API without authorisation and no request limits
+  * [Time Door](https://timedoor.io) - A time series analysis API
```

Comment:
Whats the free offering here? I do not see anything mentioned on their page. Fair enough its ridiculously cheap, but still it doesnt seem free :)

Reply:
free offering is `100000ms free compute`

Reply:
Ah I see it now, thanks!